release_notes/49-release-notes.adoc

```diff
@@ -0,0 +1,478 @@
+
+|GA
+|GA
+|DEP
+
+|API token authentication for {cloud-redhat-com}^[1]^
+|DEP
+|DEP
+|DEP
+
+|Collections hierarchical implementation^[2]^
+|GA
+|GA
+|DEP 
+
+|Compliance dashboard
+|NA
+|DEP
+|DEP
+
+|`definitions.stackrox.io`
+|DEP
+|DEP
+|DEP
+
+|*Export Download Evidence as CSV* button^[3]^
+|GA
+|GA
+|REM
+
+|Google Container Registry integration^[4]^
+|DEP
+|DEP
+|DEP
+
+|GraphQL endpoints^[5]^
+|GA
+|GA
+|DEP
+
+|GraphQL image -> scan -> components and image -> scan -> components -> vulns queries^[6]^
+|GA
+|GA
+|REM
+
+|Kernel support packages and driver download functionality^[7]^
+|DEP
+|DEP
+|DEP
+
+|Manifest installation method, also called the `roxctl` CLI installation method^[8]^
+|GA
+|GA
+|DEP
+
+|Reporting of Istio vulnerabilities
+|DEP
+|DEP
+|DEP
+
+a|`roxctl` admission controller parameters: 
+
+* `--admission-controller-enforce-on-creates`
+* `--admission-controller-enforce-on-updates`
+* `--admission-controller-listen-on-creates`
+* `--admission-controller-listen-on-updates`
+* `--admission-controller-listen-on-events`
+* `--admission-controller-timeout`
+|GA
+|GA
+|DEP
+
+|StackRox Scanner
+|DEP
+|DEP
+|DEP
+
+|`/v1/clustercves/suppress` APIs^[9,10]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/clustercves/unsuppress` APIs^[9,10]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/nodecves/suppress` APIs^[9,10]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/nodecves/unsuppress` APIs^[9,10]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/summary/counts` endpoint
+|DEP
+|DEP
+|REM
+
+|Vulnerability Management (1.0) menu item^[11]^
+|DEP
+|DEP
+|DEP
+
+|Vulnerability Report Creator permission
+|DEP
+|DEP
+|DEP
+
+|===
+
+[.small]
+--
+. API token authentication is deprecated. The corresponding cloud source integration now uses service accounts for authentication.
+
+. The *Export Download Evidence as CSV* functionality from has been removed from the image, node, and platform CVE pages due to technical issues. This functionality is provided by new reporting features. For more information about creating and downloading vulnerability reports, see xref:../operating/manage-vulnerabilities/vulnerability-reporting.adoc#vulnerability-reporting[Vulnerability reporting].
+
+. The current hierarchical implementation for defining Collections is deprecated and will be replaced by a more comprehensive search-based definition in a future release.
+
+. The Google Container Registry integration is deprecated in response to the deprecation of Container Registry. You can use the Artifact Registry as a registry replacement and Scanner V4 as a scanner replacement.
++
+For more information, see link:https://cloud.google.com/artifact-registry/docs/transition/transition-from-gcr[Transition from Container Registry] (Google Cloud documentation).
+
+. Kernel support packages and driver download functionality are deprecated.
+
+. All GraphQL endpoints are deprecated and will be removed in a future release. The endpoints were created to support the {product-title-short} portal. All other uses are unsupported. 
```

Comment:
Same comment about future plans:
"...are deprecated and are expected to be removed in a future release."